### ISSUE TITLE
[cosmos-proxy] The proxy fails finding the cache dump file

### ISSUE THREAD

**Issue by frbattid** [telefonicaid/fiware-cosmos#193]
The file existing check is failing for me. I've used:

```
function fileExists() {
    var file = fs.readFileSync(pathToFile);

    if (file) {
        return true;
    } else {
        return false;
    }
} // fileExists
```

Instead of current:

```
function fileExists() {
    try {
        fs.accessSync(pathToFile);
        return true;
    } catch (e) {
        return false;
    } // try catch
} // fileExists
```

**Comment by pcoello25** [comment]
Implemented in PR https://github.com/telefonicaid/fiware-cosmos/pull/195

**Comment by pcoello25** [comment]
Reopened due to a failed assignation between issue and PR

**Comment by pcoello25** [comment]
Implemented in PR #197
